draft-kazuho-httpbis-priority.md

```diff
@@ -300,6 +302,84 @@ receipt, a client that supports header-based prioritization MUST close the
 connection with a protocol error. Non-supporting clients will ignore this
 extension element (see {{!RFC7540}}, Section 5.5).
 
+# Reprioritization
+
+Once a client sends a request with a priority, circumstances might change and
+mean that it is beneficial to change the priority of the response. As an
+example, a web browser might issue a prefetch request for an HTML document with
+the urgency parameter of the Priority request header field set to `background`.
+Then, when the user navigates to the HTML while prefetch is in action, it would
+send a reprioritization frame with the priority field value set to `urgency=0`.
+
+However, a client cannot reprioritize a response by using the Priority header
+field.  This is because an HTTP header field can only be sent as part of an HTTP
+message. Therefore, to support reprioritization, it is necessary to define a
+HTTP-version-dependent mechanism for transmitting the priority parameters.
+
+This document specifies a new frame type for HTTP/2 ({{?RFC7540}}) and HTTP/3
+({{?I-D.draft-ietf-quic-http-23}}) that is specialized for reprioritization. It
+carries updated priority parameters and is sent on the version-specific control
+stream. The frame references the response to reprioritize based on a
```

Comment:
I think we might want to delete "and is sent on the version-specific control stream."

HTTP/2 does not have a control stream. Stream ID 0 is an identifier indicating that the frame does not belong to *any* stream.

Reply:
good point I agree. I'd like to highlight that this frame is, unlike the PRIORITY, not allowed on request streams. I'll take a stab at changing the wording around in a separate commit but if I can't get it right I will delete the offending statement you have highlighted. I have a few TODOs in the document to flesh out frame and stream handling requirements and we could defer this to them.

Reply:
Had an attempt on https://github.com/kazuho/draft-kazuho-httpbis-priority/pull/51/commits/34600ed80d5242a536b6b16b403337dce85d3a77